src/components/Identity/customerIds/createCustomerIds.js

```diff
@@ -0,0 +1,88 @@
+import { normalizeCustomerIds, validateCustomerIds } from "../util";
+import {
+  crc32,
+  convertBufferToHex,
+  convertStringToSha256Buffer
+} from "../../../utils";
+import { COOKIE_NAMES } from "../constants";
+import createEvent from "../../DataCollector/createEvent";
```

Comment:
I don't think we should have dependencies across components.  Perhaps we should move createEvent into core?

Reply:
I agree with this.

Reply:
I agree!

Reply:
Considering we are planning on not using an event at all once the new APIs open up (that's my understanding anyway), I don't mind keeping this as it is for now if you want. IMO it would be nice if we can later keep DataCollector the only component that creates events.